src/components/CytoscapeGraph/CytoscapeToolbar.tsx

```diff
@@ -183,11 +186,12 @@ export class CytoscapeToolbar extends React.PureComponent<CytoscapeToolbarProps,
               id="toolbar_layout2"
               aria-label="Graph Layout Style 2"
               className={buttonStyle}
-              variant="plain"
+              isActive={this.props.layout.name === ColaGraph.getLayout().name}
+              isDisabled={this.props.disabled}
               onClick={() => {
-                this.props.setLayout(ColaGraph.getLayout());
+                this.setLayout(ColaGraph.getLayout());
```

Comment:
Was calling directly `this.props.setLayout` doing a re-render?
If true, something may be weird with redux, because an unchanged redux state should not trigger the re-render :-/

Reply:
Not really sure about the many pieces here, but [ColaGraph.getLayout](https://github.com/kiali/kiali-ui/blob/master/src/components/CytoscapeGraph/graphs/ColaGraph.ts#L3-L10) always returns a new object (even if has the same contents). Probably should put that in a variable and return it.

Reply:
@israel-hdez It's because `ColaGraph.getLayout()` returns a new object every time.  I could have changed that behavior but instead just made this change.

Reply:
Oh, right, @josejulio just said the same thing :-)

Reply:
OK, understood :-)